### PR TITLE
HPCC-30571 LogAccess to report procid by default

### PR DESCRIPTION
**Pull request by rpastrana**
- Introduces logaccess column alias concept
- Adds WsLogAccess.getinfo Alias column
- Populates Alias column if reported in configuration
- Annotates ALA v1 instance with hpcc_log_component name
- Adds alias to helm schema
Signed-off-by: Rodrigo Pastrana <rodrigo.pastrana@lexisnexisrisk.com>

## Type of change:
- [x] This change is a bug fix (non-breaking change which fixes an issue).
- [x] This change is a new feature (non-breaking change which adds functionality).
- [ ] This change improves the code (refactor or other change that does not change the functionality)
- [ ] This change fixes warnings (the fix does not alter the functionality or the generated code)
- [ ] This change is a breaking change (fix or feature that will cause existing behavior to change).
- [ ] This change alters the query API (existing queries will have to be recompiled)

## Checklist:
- [x] My code follows the code style of this project.
  - [ ] My code does not create any new warnings from compiler, build system, or lint.
- [x] The commit message is properly formatted and free of typos.
  - [ ] The commit message title makes sense in a changelog, by itself.
  - [ ] The commit is signed.
- [ ] My change requires a change to the documentation.
  - [ ] I have updated the documentation accordingly, or...
  - [ ] I have created a JIRA ticket to update the documentation.
  - [ ] Any new interfaces or exported functions are appropriately commented.
- [x] I have read the CONTRIBUTORS document.
- [ ] The change has been fully tested:
  - [ ] I have added tests to cover my changes.
  - [ ] All new and existing tests passed.
  - [ ] I have checked that this change does not introduce memory leaks.
  - [ ] I have used Valgrind or similar tools to check for potential issues.
- [ ] I have given due consideration to all of the following potential concerns:
  - [ ] Scalability
  - [ ] Performance
  - [ ] Security
  - [ ] Thread-safety
  - [ ] Cloud-compatibility
  - [ ] Premature optimization
  - [ ] Existing deployed queries will not be broken
  - [ ] This change fixes the problem, not just the symptom
  - [ ] The target branch of this pull request is appropriate for such a change.
- [x] There are no similar instances of the same problem that should be addressed
  - [ ] I have addressed them here
  - [ ] I have raised JIRA issues to address them separately
- [ ] This is a user interface / front-end modification
  - [ ] I have tested my changes in multiple modern browsers
  - [ ] The component(s) render as expected

## Smoketest:
- [ ] Send notifications about my Pull Request position in Smoketest queue.
- [ ] Test my draft Pull Request.

### CODE REVIEW

**Review by jeclrsg — APPROVED** [review]

**Review by wangkx — APPROVED** [review]
@rpastrana please see 2 questions.

File: helm/examples/azure/log-analytics/loganalytics-hpcc-logaccess.yaml [hunk 1]
```diff
@@ -59,7 +59,7 @@ global:
     - type: "instance"
       storeName: "ContainerInventory"
       searchColumn: "Name"
-      projectName: "hpcc_log_component"
```
[the remark this answers is not in the document]
Comment (rpastrana) [reply helm/examples/azure/log-analytics/loganalytics-hpcc-logaccess.yaml:62]: Great catch @wangkx !

@jeclrsg This means we already had a mechanism in place, where I would report the Alias/ProjectName as the Name of the field so the UI doesn't have to do the lookup. I say we scrap the Alias and stick to the existing mechanism, agree?

**Comment by rpastrana** [comment]
@jeclrsg as discussed offline, these changes are not needed, and will reuse this pr to report the procid instead. Please test and let me know.

**Comment by ghalliday** [comment]
@rpastrana please can you update the jira to indicate what this is including and why it is useful. Thanks.

**Comment by rpastrana** [comment]
> @rpastrana please can you update the jira to indicate what this is including and why it is useful. Thanks.

Sure, I had already updated the Jira: "LogAccess should report procid by default"
I'll mention that the UI is expecting the procid, and @jakesmith confirmed it is useful even in containerized mode.

**Comment by rpastrana** [comment]
@wangkx the gist of this PR has changed, there's not much ESP related work but please review

**Review by jeclrsg — APPROVED** [review]

**Review by wangkx — APPROVED** [review]
@rpastrana looks fine for me.

**Comment by rpastrana** [comment]
@ghalliday ready to merge

**Pull request merged**